src/biome/text/api_new/model.py

```diff
@@ -50,6 +50,11 @@ def forward(
         embeddings = self._embedder(text, num_wrapping_dims=num_wrapping_dims)
         return self.encoder(embeddings, mask=mask)
 
+    def extend_vocab(self, vocab: Vocabulary, **kwargs):
+        """This method is automatically called when a base model updates the vocabulary"""
+        self.vocab = vocab
+        self._embedder = self.featurizer.build_embedder(self.vocab)
```

Comment:
This will keep the weights of existing embeddings and extend the size of embedding layers, right?

Reply:
This will reload the embedding layer with new vocab, so, every embedding configurations will be the same. 

Maybe we can use `Model.extend_embedder_vocab` in that case